### PR TITLE
💄(course_glimpse) fix style with narrow org logo

### LINKED ISSUE #1654: Course glimpse with incorrect style when Organization logo is too narrow
## Bug Report
When the organization logo is too narrow, the course glimpse is having the organization logo cropped.

**Screenshot**
For with an organization with an image that has bigger height and width. 
image

**Possible Solution**
Add a `max-height: 100%` on the organization logo on course glimpse.

I would send a PR with this fix suggestion.

### PR DESCRIPTION
**Pull request by igobranco**
Fix course glimpse layout when organization logo is too narrow
Closes #1654

### CODE REVIEW

**Comment by igobranco** [comment]
@jbpenrath can you review this PR ?

File: src/frontend/scss/objects/_course_glimpses.scss [hunk 1]
```diff
@@ -264,6 +264,10 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
     top: -3.53rem;
     width: 4.3rem;
     z-index: 0;
+
+    img {
+      max-height: 100%;
+    }
```
Comment (jbpenrath) [inline src/frontend/scss/objects/_course_glimpses.scss:270]: In order to cover all layout issues here, I think `object-fit` is a better workaround. What do you think?

```suggestion
    img {
      object-fit: contain;
      object-position: center;
    }
```

Comment (igobranco) [reply src/frontend/scss/objects/_course_glimpses.scss]: I didn't know that property...

To make it work, the `img` tag also needs an explicit `height` to have the same effect.
```css
    img {
      object-fit: contain;
      object-position: center;
      height: 4rem;
    }
```

To cover other layout issues, I think it is better to include the width.
```css
    img {
      object-fit: contain;
      object-position: center;
      height: 4rem;
      width: 4rem;
    }
```

File: src/frontend/scss/objects/_course_glimpses.scss [hunk 2]
```diff
@@ -264,6 +264,13 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
     top: -3.53rem;
     width: 4.3rem;
     z-index: 0;
+
+    img {
+      object-fit: contain;
+      object-position: center;
+      height: 4rem;
+      width: 4rem;
```
Comment (jbpenrath) [inline src/frontend/scss/objects/_course_glimpses.scss:272]: You could use percentage value instead to force image to fill its container. Just more maintainable.
```suggestion 
      height: 100%;
      width: 100%;
```

**Review by jbpenrath — APPROVED** [review]
Just change width and height values.
Otherwise looks good to me ✅

**Pull request merged**